code/game/objects/items/stacks/telecrystal.dm

```diff
@@ -34,8 +34,10 @@
 		use(amount)
 		to_chat(user, "<span class='notice'>You slot [src] into [cart].  The next time it's used, it will also give telecrystals</span>")
 
-/obj/item/stack/telecrystal/detailed_examine_antag()
-	return "Telecrystals can be activated by utilizing them on devices with an actively running uplink. They will not activate on unactivated uplinks."
+/obj/item/stack/telecrystal/examine(mob/user)
+	. = ..()
+	if(isAntag(user))
```

Comment:
To anyone curious, isAntag returns /all/ special roles, so stuff like ERT can see these texts

In the detailed_examine_antag(), this is also the case
Yes, prepared to get outed if you have power gloves in your bag by spiders in a swat suit